mkdocs.yml

```diff
@@ -43,3 +43,5 @@ nav:
     - 'OpenShift and Container Storage for Admins': 'admin-storage.md'
     - 'OpenShift Metering': 'metering.md'
     - 'OpenShift Virtualization': 'openshift-virtualization.md'
+- Security:
+    - 'Cyberark Secrets Management for OpenShift Workshop': 'cyberark-for-openshift.md'
```

Comment:
Please move the Black Duck content into this section, since it is also security-related. We'll end up having to split hairs on what is "appdev" and what is "security", but it's probably fine for now to just lump all the security things into this new section.

Reply:
@davemeurer the linewrap got fixed in the most recent PR but they didn't move the other content into the Security section.

Reply:
hmmm...it's moved in my local version. I'll try again.